apps/main/src/loan/components/PageLlamaMarkets/Page.tsx

```diff
@@ -61,6 +63,8 @@ export const LlamaMarketsPage = (props: CrvUsdServerData) => {
   useInjectServerData(props)
   const { signerAddress } = useWallet()
   const { data, isError, isLoading } = useLlamaMarkets(signerAddress)
+  const minLiquidity = useUserProfileStore((s) => s.hideSmallPools) ? SMALL_POOL_VOLUME : 0
```

Comment:
would it be nicer to have a getter function in the store that returns minLiquidity directly?

`const minLiquidity = useUserProfileStore((s) => s.minLiquidity)` where minLiquidity is a small getter function that returns the value of `hideSmallPools ? SMALL_POOL_VOLUME : 0`.

Benefit would be not having to import and perform manual logic with `SMALL_POOL_VOLUME`
Downside would be added complexity to the store, even if it's such a simple getter.

I think the benefit outweighs the downside but it's not a blocker, just a question of preference.

Reply:
Yeah I had the same thought now that they are based on the same constant. I'll fix this in https://github.com/curvefi/curve-frontend/pull/849

Reply:
I remember now why I didn't do it :laughing: this constant is only used with the store here.
The other apps using the constant are adding it to the chain configuration